[2025-03-17T22:30:20.191+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-16T17:57:48.090379+00:00 [queued]>
[2025-03-17T22:30:20.320+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-16T17:57:48.090379+00:00 [queued]>
[2025-03-17T22:30:20.321+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-17T22:30:20.853+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): transform_data_phase_1_task> on 2025-03-16 17:57:48.090379+00:00
[2025-03-17T22:30:20.863+0000] {standard_task_runner.py:60} INFO - Started process 15873 to run task
[2025-03-17T22:30:20.935+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'Google_map', 'transform_data_phase_1_task', 'manual__2025-03-16T17:57:48.090379+00:00', '--job-id', '191', '--raw', '--subdir', 'DAGS_FOLDER/google_map_dag_etl.py', '--cfg-path', '/tmp/tmp4j6w_3u7']
[2025-03-17T22:30:20.945+0000] {standard_task_runner.py:88} INFO - Job 191: Subtask transform_data_phase_1_task
[2025-03-17T22:30:21.933+0000] {task_command.py:423} INFO - Running <TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-16T17:57:48.090379+00:00 [running]> on host DESKTOP-AVAHALM.
[2025-03-17T22:30:25.574+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='master_m2si' AIRFLOW_CTX_DAG_ID='Google_map' AIRFLOW_CTX_TASK_ID='transform_data_phase_1_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T17:57:48.090379+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T17:57:48.090379+00:00'
[2025-03-17T22:30:25.577+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-17T22:30:25.578+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd ~/data_warehouse_project && dbt run --profiles-dir ~/.dbt --models cleaned_reviews']
[2025-03-17T22:30:25.636+0000] {subprocess.py:86} INFO - Output:
[2025-03-17T22:30:43.805+0000] {subprocess.py:93} INFO - [0m22:30:43  Running with dbt=1.9.0-b2
[2025-03-17T22:30:48.546+0000] {subprocess.py:93} INFO - [0m22:30:48  Registered adapter: postgres=1.8.2
[2025-03-17T22:30:51.294+0000] {subprocess.py:93} INFO - [0m22:30:51  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-17T22:30:51.295+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-03-17T22:30:51.295+0000] {subprocess.py:93} INFO - - models.data_warehouse_project.example
[2025-03-17T22:30:54.138+0000] {subprocess.py:93} INFO - [0m22:30:54  Found 6 models, 1 source, 424 macros
[2025-03-17T22:30:54.154+0000] {subprocess.py:93} INFO - [0m22:30:54
[2025-03-17T22:30:54.189+0000] {subprocess.py:93} INFO - [0m22:30:54  Concurrency: 4 threads (target='dev')
[2025-03-17T22:30:54.193+0000] {subprocess.py:93} INFO - [0m22:30:54
[2025-03-17T22:30:57.900+0000] {subprocess.py:93} INFO - [0m22:30:57  1 of 1 START sql table model public.cleaned_reviews ............................ [RUN]
[2025-03-17T22:31:02.192+0000] {subprocess.py:93} INFO - [0m22:31:02  1 of 1 OK created sql table model public.cleaned_reviews ....................... [[32mSELECT 7[0m in 4.21s]
[2025-03-17T22:31:02.637+0000] {subprocess.py:93} INFO - [0m22:31:02
[2025-03-17T22:31:02.651+0000] {subprocess.py:93} INFO - [0m22:31:02  Finished running 1 table model in 0 hours 0 minutes and 8.44 seconds (8.44s).
[2025-03-17T22:31:03.289+0000] {subprocess.py:93} INFO - [0m22:31:03
[2025-03-17T22:31:03.303+0000] {subprocess.py:93} INFO - [0m22:31:03  [32mCompleted successfully[0m
[2025-03-17T22:31:03.304+0000] {subprocess.py:93} INFO - [0m22:31:03
[2025-03-17T22:31:03.319+0000] {subprocess.py:93} INFO - [0m22:31:03  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-03-17T22:31:08.227+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-17T22:31:10.719+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Google_map, task_id=transform_data_phase_1_task, execution_date=20250316T175748, start_date=20250317T223020, end_date=20250317T223110
[2025-03-17T22:31:13.564+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-17T22:31:15.078+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
